### ISSUE TITLE
Get a UserWarning: Implicit dimension choice for softmax when training mini-RTS following Install tutorial.

### ISSUE THREAD

**Issue by caidwang** [facebookresearch/ELF#118]
I get a User warning when start the training demo `sh ./train_minirts.sh` in the environment with python 3.6 and pytorch-cpu 0.3.1. It looks like this:
![2018-11-11 11-02-13](https://user-images.githubusercontent.com/20463134/48308566-1932f500-e5a3-11e8-81bf-3b2fe0a1d8e6.png)
The warning seems to be not serious but I think it's a little confusing which dimension the softmax is working on (I guess it might be dim=0 ?). Whatever, I just suggest that this warning can be cleaned up by specifying the dimension exacter.
